src/test/kotlin/org/jetbrains/research/ml/dataset/anonymizer/util/AnonymizerTest.kt

```diff
@@ -30,18 +31,32 @@ open class AnonymizerTest(testDataRoot: String) : ParametrizedBaseTest(testDataR
     @Parameterized.Parameter(1)
     var outFile: File? = null
 
-    protected fun assertCodeAnonymization(inFile: File, outFile: File, anonymizer: Anonymizer) {
+    private fun getFragments(df: DataFrame): List<String> {
+        if (Column.FRAGMENT.key !in df.cols.map { it.name }) {
+            return emptyList()
+        }
+        return (df[Column.FRAGMENT.key] as StringCol).values.mapNotNull { it }
+    }
+
+    // Usually we have to create psi file from fixture to correct find all references.
+    // But for Kotlin we have to create temporary files instead it
```

Comment:
instead it -> you can use just **instead**, without **it** (more preferable I think), or **instead of it**